src/Microsoft.OpenApi/Services/OpenApiFilterService.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.Services
+{
+    /// <summary>
+    /// A service that slices an OpenApiDocument into a subset document
+    /// </summary>
+    public static class OpenApiFilterService
+    {
+        /// <summary>
+        /// Create predicate function based on passed query parameters
+        /// </summary>
+        /// <param name="operationIds">Comma delimited list of operationIds or * for all operations.</param>
+        /// <returns>A predicate.</returns>
+        public static Func<OpenApiOperation, bool> CreatePredicate(string operationIds)
+        {
+            string predicateSource = null;
+
+            Func<OpenApiOperation, bool> predicate;
+            if (operationIds != null)
+            {
+                if (operationIds == "*")
+                {
+                    predicate = (o) => true;  // All operations
+                }
+                else
+                {
+                    var operationIdsArray = operationIds.Split(',');
+                    predicate = (o) => operationIdsArray.Contains(o.OperationId);
+                }
+
+                predicateSource = $"operationIds: {operationIds}";
```

Comment:
Is this variable meant to be used somewhere?

Reply:
Its unused actually. Let me resolve it.